cfme/common/vm.py

```diff
@@ -957,3 +957,12 @@ def __init__(self, name, provider, template_name=None):
     def does_template_exist_on_provider(self):
         """Check if template exists on provider itself"""
         return self.provider.mgmt.does_template_exist(self.name)
+
+
+def cleanup_vm(vm_name, provider):
+    try:
+        logger.info('Cleaning up VM %s on provider %s', vm_name, provider.key)
+        provider.mgmt.delete_vm(vm_name)
+    except:
+        # The mgmt_sys classes raise Exception :\
+        logger.warning('Failed to clean up VM %s on provider %s', vm_name, provider.key)
```

Comment:
Optional: This code can be used in https://github.com/ManageIQ/integration_tests/blob/master/cfme/fixtures/vm_name.py#L14
also .

Reply:
thanks but  this is small part of huge task and I don't want to deviate because each change requires much time for testing.

Reply:
:+1: